pubspec.yaml

```diff
@@ -101,7 +103,6 @@ flutter:
   uses-material-design: true
 
   assets:
-    - .env
```

Comment:
we won't need this? (or need a rebase from main)

Reply:
rebased and made sure the versions matched